addons/danger/functions/fnc_tacticsSuppress.sqf

```diff
@@ -65,8 +65,9 @@ _pos pushBack _target;
 _unit setVariable [QEGVAR(main,currentTarget), _target, EGVAR(main,debug_functions)];
 _unit setVariable [QEGVAR(main,currentTask), "Leader Suppress", EGVAR(main,debug_functions)];
 
-private _group = group _unit;
 // set group task
+private _group = group _unit;
+_group enableAttack false;
```

Comment:
when will this be set to true again?

Reply:
L39 of same file.  When the reset is configured.  Actually toggling attack mode was missing.